INSTALL.md

```diff
@@ -111,7 +111,7 @@ In order to install the llvm (clang) toolchain, you may have to follow instructi
 When building on OSX, here's some dependencies you'll need:
 - Install xcode
 - Install [homebrew](https://brew.sh)
-- `brew install libsodium libtool autoconf automake pkg-config libpq openssl parallel ccache bison sed perl`
+- `brew install libsodium libtool autoconf automake pkg-config libpq openssl parallel ccache bison sed perl coreutils`
```

Comment:
there is no `sed` package. using `gnu-sed` instead.

Reply:
updated